integrationtests/Paket.IntegrationTests/PackSpecs.fs

```diff
@@ -349,3 +349,19 @@ let ``#1596 pack works for reflected definition assemblies``() =
     
     File.Exists(Path.Combine(outPath, "lib", "net45", "Project2.dll")) |> shouldEqual true
     File.Delete(templatePath)
+
+[<Test>]
+let ``#1806 pack localized dll`` () =
+    let scenario = "i001806-pack-localized"
+    let rootPath = scenarioTempPath scenario
+    let outPath = Path.Combine(rootPath, "out")
+    let package = Path.Combine(outPath, "LocalizedLib.1.0.0.0.nupkg")
+    
+    paket ("pack -v output \"" + outPath) scenario |> ignore
+    ZipFile.ExtractToDirectory(package, outPath)
+
+    Path.Combine(outPath, "lib", "net45", "LocalizedLib.dll") |> checkFileExists
```

Comment:
Yes, it's minor thing.

That same operator is defined in FAKE and pretty much any codebase dealing with paths so I thin it is not too far from reach for average F# developer.

You can use go to definition on operators also :smile: